reactor-core/src/main/java/reactor/core/publisher/Hooks.java

```diff
@@ -465,6 +468,35 @@ public static void resetOnNextError() {
 		}
 	}
 
+	@SuppressWarnings("unchecked")
+	static <T> CorePublisher<T> onLastOperator(CorePublisher<T> source) {
```

Comment:
maybe rename to `assembleLastOperator`? it might get too confusing with the `onLastOperatorHook` field and `onLastOperator(Function)` setter. Another option is to move that part to `Operators` rather than `Hooks` class (the later being used to get and set the hooks, the former to apply the hooks and potentially find local overrides in the `Context`)